package.json

```diff
@@ -44,7 +44,7 @@
     "@ungap/custom-elements": "^0.1.15",
     "lit": "^2.1.2",
     "swipe-detect": "^1.0.1",
-    "vl-mapactions": "^4.0.1"
+    "vl-mapactions": "^4.1.0"
```

Comment:
https://uig-webcomponents.omgeving.vlaanderen.be/?path=/story/custom-elements-vl-map-vl-map-modify-action--map-with-polygon-modify Als je deze story bekijkt is er geen minimap. Als ik lokaal deze pr run, krijgt die story een minimap zonder kaart er in (bug), ergens iets misgelopen?

Reply:
Een bugje in de vl-mapactions: https://github.com/milieuinfo/vl-mapactions/pull/64